src/main/java/com/redhat/devtools/intellij/lsp4ij/operations/completion/LSIncompleteCompletionProposal.java

```diff
@@ -185,12 +181,15 @@ public String getLookupString() {
     }
 
     private boolean isDeprecated() {
-        return item.getDeprecated() != null && item.getDeprecated().booleanValue();
+        return (item.getTags() != null && item.getTags().contains(CompletionItemTag.Deprecated))
+        || (item.getDeprecated() != null && item.getDeprecated().booleanValue());
     }
 
     @Override
     public void renderElement(LookupElementPresentation presentation) {
         presentation.setItemText(item.getLabel());
+        presentation.setTypeText(item.getDetail());
```

Comment:
item detail now added to the type text:

<img width="681" alt="Screenshot 2023-06-29 at 15 22 50" src="https://github.com/redhat-developer/intellij-quarkus/assets/148698/497308fe-669d-4b84-9948-6c2a20a4ddd5">
<img width="1020" alt="Screenshot 2023-06-29 at 15 25 00" src="https://github.com/redhat-developer/intellij-quarkus/assets/148698/567e8748-b9eb-4b0e-badf-a69a1980378b">